src/Operation/FindOneAndUpdate.php

```diff
@@ -59,9 +59,9 @@ public function __construct($databaseName, $collectionName, $filter, $update, ar
             throw new InvalidArgumentTypeException('$update', $update, 'array or object');
         }
 
-        if ( ! \MongoDB\is_first_key_operator($update)) {
-            throw new InvalidArgumentException('First key in $update argument is not an update operator');
-        }
+      //  if ( ! \MongoDB\is_first_key_operator($update)) {
+       //     throw new InvalidArgumentException('First key in $update argument is not an update operator');
```

Comment:
Oh, I just realized this was modifying an Operation class. We should talk tomorrow about why this was necessary; however, it may no longer be relevant if you'll end up using `executeBulkWrite()` directly.